### DISCUSSION TITLE
Environment Variables

### DISCUSSION THREAD

**Discussion by mycbdsolution** [bigcommerce/catalyst#1784] [General]
Did the environment variables change recently? I’ve been working on the same fork for a couple months and it uses “customer impersonation token” but I recently made a new fork and the environment variables seem slightly different. Most notable being “storefront token” replacing the customer token. The documentation is confusing and doesn’t seem to match what I’m seeing. Appreciate any help.

**Comment by matthewvolk** [comment]
Hi @mycbdsolution! We have a description of the changes in our release notes detailed here: https://github.com/bigcommerce/catalyst/releases/tag/%40bigcommerce%2Fcatalyst-core%400.19.0

TL;DR you can run `npx @bigcommerce/create-catalyst init` to generate updated environment variables (including the storefront token)

We'll take a look at the documentation to make sure everything lines up, great callout. Let us know if you still have any questions!

**Reply by mycbdsolution** [reply]
Got it. Thank you for your help!
